plugins/woocommerce/includes/class-wc-webhook.php

```diff
@@ -606,7 +615,18 @@ public function deliver_ping() {
 			'body'       => 'webhook_id=' . $this->get_id(),
 		);
 
-		$test          = wp_safe_remote_post( $this->get_delivery_url(), $args );
+		/**
+		 * Filters the HTTP arguments for the webhook delivery ping.
+		 *
+		 * @since 3.3.0
```

Comment:
Not sure if we have precedence for this, but this filter has been around since 3.3.0, but only applied to delivery pings in 10.3.0.